### PR TITLE
shortsubject: skip when commit is a merge commit

### PR DESCRIPTION
**Pull request by tmrts**
Fixes https://travis-ci.org/coreos/rkt/jobs/132579478

cc @lucab

### CODE REVIEW

File: rules/shortsubject/shortsubject.go [hunk 1]
```diff
@@ -21,11 +23,12 @@ func init() {
...
-	if len(c["subject"]) >= 90 {
+	if len(c["subject"]) >= 90 && len(strings.Split(c["parent"], " ")) <= 1 {
 		vr.Pass = false
 		vr.Msg = "commit subject exceeds 90 characters"
 		return
 	}
+
 	vr.Pass = true
 	if len(c["subject"]) > 72 {
 		vr.Msg = "commit subject is under 90 characters, but is still more than 72 chars"
```
Comment (jonboulle) [inline rules/shortsubject/shortsubject.go:40]: this is no longer necessary true. could do with another conditional branch dealing with the new case

Comment (tmrts) [inline rules/shortsubject/shortsubject.go:40]: done PTAL

File: rules/shortsubject/shortsubject.go [hunk 2]
```diff
@@ -21,11 +23,18 @@ func init() {
 // ValidateShortSubject checks that the commit's subject is strictly less than
 // 90 characters (preferably not more than 72 chars).
 func ValidateShortSubject(c git.CommitEntry) (vr validate.Result) {
+	if len(strings.Split(c["parent"], " ")) <= 1 {
```
Comment (jonboulle) [inline rules/shortsubject/shortsubject.go]: `> 1`

Comment (tmrts) [inline rules/shortsubject/shortsubject.go]: damn it

Comment (tmrts) [inline rules/shortsubject/shortsubject.go]: PTAL

**Pull request merged**